#! /bin/sh /usr/share/dpatch/dpatch-run
##  by Ludovic Rousseau <rousseau@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: correct makefile

@DPATCH@
--- a/makefile
+++ b/makefile
@@ -14,3 +14,10 @@
 
 jhead: $(objs) jhead.h
 	${CC} -o jhead $(objs) -lm
+
+clean:
+	rm -f $(objs) jhead
+
+install:
+	cp jhead ${DESTDIR}/usr/bin/
+
